cli/cmd/cmdutil/cmdutil.go

```diff
@@ -90,17 +90,42 @@ func Client(cfg *config.Config) (*client.Client, error) {
 	return c, nil
 }
 
-func PromptGetSelect(items []string, label string) string {
-	prompt := promptui.Select{
-		Label: label,
-		Items: items,
+func SelectPrompt(msg string, options []string, def string) string {
```

Comment:
Just checking: if `def` is empty, it will still work, right? (Just without a default)

Reply:
Actually, there are cases where there is no default (so `def == ""`) where the user should be forced to make a choice.

Reply:
Yeah exactly so I will be checking if default value provided exists in options. If not I won't set any default in the prompt.